CHANGES.md

```diff
@@ -1,6 +1,10 @@
 History
 =========
 
+0.16.0 (2020-03-24)
+----------------
+* Added support for distinct on (#66)
```

Comment:
Didn't think of this earlier, but it'd be nice to add an example to this repo's README.

Reply:
I will add.